src/logos/payu/logo.jsx

```diff
@@ -9,22 +9,16 @@ import { type LogoColorMap } from "../../types";
 
 const LOGO_COLORS: LogoColorMap = {
   [LOGO_COLOR.DEFAULT]: {
-    primary: "#A6d71c",
-    secondary: "#3C932A",
-    tertiary: "#6CC62E",
-    quaternary: "#278D30",
+    primary: "#A6C307",
+    secondary: "#FFFFFF",
   },
   [LOGO_COLOR.WHITE]: {
     primary: "#FFFFFF",
     secondary: "#FFFFFF",
-    tertiary: "#000000",
-    quaternary: "#FFFFFF",
   },
   [LOGO_COLOR.BLACK]: {
-    primary: "#333030",
-    secondary: "#333030",
-    tertiary: "#ffffff",
-    quaternary: "#333030",
+    primary: "#A6C307",
```

Comment:
Based on the UX design, the logo rendered on the white and silver buttons should be dark. The way you have it configured shows the default branded color.
```suggestion
    primary: "#2C2E2F",
    secondary: "#2C2E2F"
```
<img width="860" alt="Screen Shot 2022-11-01 at 5 39 17 PM" src="https://user-images.githubusercontent.com/9424291/199367782-4f1e433d-ee80-4878-883c-d714c45cceaa.png">